client/object_get.go

```diff
@@ -0,0 +1,303 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	v2object "github.com/nspcc-dev/neofs-api-go/v2/object"
+	v2refs "github.com/nspcc-dev/neofs-api-go/v2/refs"
+	rpcapi "github.com/nspcc-dev/neofs-api-go/v2/rpc"
+	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	v2session "github.com/nspcc-dev/neofs-api-go/v2/session"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/nspcc-dev/neofs-sdk-go/session"
+	"github.com/nspcc-dev/neofs-sdk-go/token"
+)
+
+// PrmObjectGet groups parameters of ObjectGetInit operation.
+type PrmObjectGet struct {
+	raw bool
+
+	local bool
+
+	sessionSet bool
+	session    session.Token
+
+	bearerSet bool
+	bearer    token.BearerToken
+
+	cnrSet bool
+	cnr    cid.ID
+
+	objSet bool
+	obj    oid.ID
+}
+
+// MarkRaw marks an intent to read physically stored object.
+func (x *PrmObjectGet) MarkRaw() {
+	x.raw = true
+}
+
+// MarkLocal tells the server to execute the operation locally.
+func (x *PrmObjectGet) MarkLocal() {
+	x.local = true
+}
+
+// WithinSession specifies session within which object should be read.
+func (x *PrmObjectGet) WithinSession(t session.Token) {
```

Comment:
Don't you want to unify these parameters like `prmSession` in container requests? It seems that all object requests will have session token, bearer token, possibility to make local requests.